lms/static/sass/discussion/_discussion.scss

```diff
@@ -444,7 +444,7 @@ section.discussion-pagination {
   .response-display-count {
     display: block;
     padding: ($baseline/2) 0;
-    color: $gray;
+    color: $gray-d2;
```

Comment:
Can you change this to use a variable from variables-v1.scss (and variables-v2.scss), and if there isn't a close enough one then introduce a new variable. My goal is to have no colors used directly in the Sass, but to instead have everything defined through variables that can be overridden in a theme.